frappe/core/doctype/communication/communication.py

```diff
@@ -273,6 +273,11 @@ def on_update(self):
 		# comments count for the list view
 		update_comment_in_doc(self)
 
+		parent = get_parent_doc(self)
+		if hasattr(parent, "on_communication_update"):
```

Comment:
https://sourcegraph.com/search?q=context:global+on_communication_update+repo:%5Egithub%5C.com/frappe/.*&patternType=standard&sm=0&groupBy=repo


Seems to be unused. We should check if it's callable or not at least.